src/legacy/core_plugins/kibana/public/discover/np_ready/components/sidebar/discover_sidebar.tsx

```diff
@@ -0,0 +1,300 @@
+
+  const selectedFields = filterFieldList(groupedFields.selected, fieldFilterState, fieldCounts);
+  const popularFields = filterFieldList(groupedFields.popular, fieldFilterState, fieldCounts);
+  const unpopularFields = filterFieldList(groupedFields.unpopular, fieldFilterState, fieldCounts);
+  return (
+    <section
+      className="sidebar-list"
+      aria-label={i18n.translate(
+        'kbn.discover.fieldChooser.filter.indexAndFieldsSectionAriaLabel',
+        {
+          defaultMessage: 'Index and fields',
+        }
+      )}
+    >
+      <DiscoverIndexPattern
+        selectedIndexPattern={selectedIndexPattern}
+        setIndexPattern={setIndexPattern}
+        indexPatternList={sortBy(indexPatternList, o => o.attributes.title)}
+      />
+      <div className="dscSidebar__item">
+        <form>
+          <DiscoverFieldSearch
+            onChange={onChangeFieldSearch}
+            value={fieldFilterState.name}
+            types={fieldTypes}
+          />
+        </form>
+      </div>
+      <div className="sidebar-list">
+        {fields.length > 0 && (
+          <>
+            <EuiTitle size="xxxs" id="selected_fields">
+              <h3>
+                <FormattedMessage
+                  id="kbn.discover.fieldChooser.filter.selectedFieldsTitle"
+                  defaultMessage="Selected fields"
+                />
+              </h3>
+            </EuiTitle>
+            <ul
+              className="dscSidebarList dscFieldList--selected"
+              aria-labelledby="selected_fields"
+              data-test-subj={`fieldList-selected`}
+            >
+              {selectedFields.map((field: IndexPatternField, idx: number) => {
+                return (
+                  <li key={`field${idx}`} data-attr-field={field.name} className="dscSidebar__item">
+                    <DiscoverField
+                      field={field}
+                      indexPattern={selectedIndexPattern}
+                      onAddField={onAddField}
+                      onRemoveField={onRemoveField}
+                      onAddFilter={onAddFilter}
+                      onShowDetails={onShowDetails}
+                      getDetails={getDetailsByField}
+                      showDetails={openFieldMap.get(field.name) || false}
+                      selected={true}
+                      useShortDots={useShortDots}
+                    />
+                  </li>
+                );
+              })}
+            </ul>
+            <div className="euiFlexGroup euiFlexGroup--gutterMedium">
+              <EuiTitle size="xxxs" id="available_fields" className="euiFlexItem">
+                <h3>
+                  <FormattedMessage
+                    id="kbn.discover.fieldChooser.filter.availableFieldsTitle"
+                    defaultMessage="Available fields"
+                  />
+                </h3>
+              </EuiTitle>
+              <div className="euiFlexItem euiFlexItem--flexGrowZero">
+                <EuiButtonIcon
```

Comment:
Could you please clarify for me, what this button is needed for?
Why it's invisible?

Reply:
Of course, it's a button that's only displayed when there's not enough screen space and the sidebar is displayed on top of the data table. it's used to toggle the view of the available fields
<img width="756" alt="Bildschirmfoto 2020-03-26 um 16 19 21" src="https://user-images.githubusercontent.com/463851/77663730-af3fa380-6f7d-11ea-94f5-a1fc25f582d1.png">